projects/mainframe.md

```diff
@@ -0,0 +1,112 @@
+-   Alignment of semantic conventions for virtualization between mainframe and other virtualizations solutions, as far as feasible and reasonable
+-   Description of mapping between names and mainframe terms (for conventions not specifically defined as mainframe extensions)
+-   Definition of a curated, most meaningful set of metrics (out of the thousands of available metrics) to be fully supported by the semantic conventions.
+-   Description and definition of semantic conventions for metrics determined as result from aggregation across system topology to simplify handling in observability backends
+-   Definition of approach for supporting instrumentation of mainframe-specific programming languages
+-   Definition of approach for supporting instrumentation of mainframe-specific logs
+-   Definition of approach for supporting trace context propagation
+-   Code instrumentation (SDKs) for mainframe-specific programming languages and runtimes
+
+## Staffing / Help wanted
+
+Based on previous participation in the meeting for preparings this proposal, we foresee continued contributions from 12 to 15 people. In addition, this group will welcome contributors from various areas:
+-   We look for participation from mainframe experts of all kinds: observability and monitoring experts, performance experts, virtualization experts, architects, and users of observability solutions for the mainframe and those who want to extend, simplify, and improve their current observability solution to the mainframe.
+-   We welcome developers working with Assembler, COBOL, PL/1, and REXX, and engineers willing to contribute with the instrumentation for mainframe-specific programming languages.
+-   We invite everybody to join who is passionate about the mainframe.
+-   Two project leads who are willing to drive the project and address any issues which are not handled by other project members.
+-   At least two sponsoring TC (Technical Committee) members. TC sponsors are dedicated to attending meetings, reviewing proposals, and in general being aware of the state of the project and its technical details. TC sponsors guide the project through the spec process, keep the tracking issue up to date, and help to ensure that relevant community members provide input at the appropriate times.
```

Comment:
can you add the TC/GC sponsors to the staffing section below?

Reply:
Added GC sponsors, and statement with respect to TC support.